File: Programación Orientada a Objetos (PDOO)/Napakalaki(Java)/src/napakalaki/Cultist.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author Josele
 */
public class Cultist {
    private String name;
    private int gainedLevels;
    
    public Cultist (String nm , int glvl){
        name=nm;
        gainedLevels=glvl;
    }
    public int getGainedLevels(){
        return gainedLevels;
    }
}

File: Programación Orientada a Objetos (PDOO)/Napakalaki(Ruby)/lib/cultist.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
module NapakalakiGame
class Cultist
  attr_accessor :name
  attr_accessor :gained_levels
  def initialize(nm,glvl)
    @name=nm
    @gained_levels=glvl
  end
end
end